Zend/zend_vm_def.h

```diff
@@ -8755,6 +8755,74 @@ ZEND_VM_HANDLER(200, ZEND_FETCH_GLOBALS, UNUSED, UNUSED)
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HOT_HANDLER(202, ZEND_SEND_PARTIAL, UNUSED, CONST|UNUSED|NUM)
+{
+    USE_OPLINE
+    zval *arg;
+    zend_execute_data *call = EX(call);
+
+    if (OP2_TYPE == IS_CONST) {
+        SAVE_OPLINE();
+        
+        uint32_t arg_num = 0;
+        zend_string *arg_name =
+            Z_STR_P(RT_CONSTANT(opline, opline->op2));
+
+	    arg = zend_handle_named_arg(
+	        &call, arg_name, &arg_num, CACHE_ADDR(opline->result.num));
+
+        if (!arg) {
+            FREE_OP2();
+            HANDLE_EXCEPTION();
+        }
+    } else {
+        arg = ZEND_CALL_ARG(call, opline->op2.num);
+    }
+    
+    ZVAL_UNDEF(arg);
+    
+    ZEND_ADD_CALL_FLAG(EX(call), ZEND_CALL_PARTIAL);
+    
+    ZEND_VM_NEXT_OPCODE();
+}
+
+ZEND_VM_HANDLER(203, ZEND_PARTIAL_APPLY, ANY, ANY)
+{
+    USE_OPLINE
+    zend_execute_data *call = EX(call);
+    zval *result;
+
+    if ((call->func->type == ZEND_INTERNAL_FUNCTION &&
```

Comment:
The pass case, for no-op ctors

Reply:
Is it actually necessary to handle this case specially, or is it just an optimization?